packages/synapse-react-client/src/components/DownloadCart/DownloadCartPage.tsx

```diff
@@ -129,22 +129,39 @@ export const DownloadCartPage: React.FunctionComponent<
           </ul>
         </div>
       </div>
-      {selectedTabIndex == 0 && !isError && !isLoading && data && (
-        <div>
-          {data.numberOfFilesRequiringAction > 0 && (
+
+      <div
+        style={{
+          /*
+           The DownloadListActionsRequired component stores actions in local state to track which actions are completed.
+
+           Mount the actions required component and hide it with style to ensure actions stored in component state are persisted as the user switches tabs
+           */
+          display:
+            selectedTabIndex == 0 && !isError && !isLoading && data
+              ? 'block'
+              : 'none',
+        }}
```

Comment:
Allow switching between the `Actions Required` and `Download List` tabs without clearing the local state in `Actions Required`